Resources/templates/responsive/workshop/partials/partner_footer.php

```diff
@@ -0,0 +1,13 @@
+<div class="partner">
+	<div class="container">
+		<div class="row">
+			<?php foreach($this->footer_sponsors as $sponsor): ?>
+				<div class="col-xs-12">
+					<a target="_blank" href="<?= $sponsor->url ?>" class="center-block">
+	            		<img class="img-responsive no-title center-block" src="<?= $sponsor->getImage()->getLink(1200, 0, false) ?>" alt="<?= $sponsor->name ?>">
+	        		</a>
```

Comment:
Here, for future developments, we could lazy load these images and also add them using a picture tag so that we can make the browser download only the image size that it's going to display.

Reply:
Yes, we can take into account lazy load in the future.